### ISSUE TITLE
Define 'save_feature_label' in 'session_data_new.py'

### ISSUE THREAD

**Issue by jeff1evesque** [jeff1evesque/machine-learning#1357]
After #1354, and #1358 has been resolved, we will define the `save_feature_label` method in `session_data_new.py`.  This method will be responsible for passing necessary arguments into `data_saver.py`, such that feature labels (independent variables) are saved into `tbl_feature_label` database table.

**Comment by jeff1evesque** [comment]
After we have resolved #1358, we will specifically implement `get_feature_labels` method within `session_data_new.py` for both *xml*, and *csv* file uploads

**Comment by jeff1evesque** [comment]
We need to fix each typo instance:

```python
...
            # check label consistency, append label(s) to 'feature_labels'
            if sorted(dataset_converter.get_feature_labels()) == self.feature_labels: self.response_error.append('The supplied features (independent variables) are inconsistent')
            self.feature_labels = sorted(dataset_converter.get_feature_labels())
...
```

to the following:

```python
...
            # check label consistency, append label(s) to 'feature_labels'
            if sorted(dataset_converter.get_feature_labels()) == self.feature_labels: self.response_error.append('The supplied observations (dependent variables) are inconsistent')
            self.feature_labels = sorted(dataset_converter.get_feature_labels())
...
```
